core/src/main/java/hera/core/commands/Config.java

```diff
@@ -0,0 +1,82 @@
+package hera.core.commands;
+
+import discord4j.core.event.domain.message.MessageCreateEvent;
+import discord4j.core.object.entity.Guild;
+import discord4j.core.object.entity.Member;
+import discord4j.core.object.entity.MessageChannel;
+import hera.core.HeraUtil;
+import hera.core.messages.MessageHandler;
+import hera.core.messages.MessageSpec;
+import hera.database.entities.ConfigFlag;
+import hera.database.entities.ConfigFlagType;
+import hera.database.entities.Localisation;
+import hera.database.types.ConfigFlagName;
+import hera.database.types.LocalisationKey;
+import reactor.core.publisher.Mono;
+
+import java.util.List;
+import java.util.concurrent.atomic.AtomicBoolean;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import static hera.core.HeraUtil.LOG;
+import static hera.store.DataStore.STORE;
+
+public class Config {
+	public static Mono<Void> execute(MessageCreateEvent event, Guild guild, Member member, MessageChannel channel, List<String> params) {
+		Localisation local = HeraUtil.getLocalisation(LocalisationKey.COMMAND_CONFIG, guild);
+		Localisation localEnabled = HeraUtil.getLocalisation(LocalisationKey.COMMON_ENABLED, guild);
+		Localisation localDisabled = HeraUtil.getLocalisation(LocalisationKey.COMMON_DISABLED, guild);
+
+		if (params.size() == 0 || params.get(0).equals("")) {
+			// list all config flags and their current value
+			StringBuilder message = new StringBuilder();
+			List<ConfigFlag> flags = STORE.configFlags().forGuild(guild.getId().asLong());
+
+			Stream.of(ConfigFlagName.values()).forEach(flag -> {
+				message.append(flag.toString().toLowerCase());
+				message.append(": ");
+				List<ConfigFlag> guildSetFlag = flags.stream().filter(f -> f.getConfigFlagType().getName() == flag).collect(Collectors.toList());
+				if (!guildSetFlag.isEmpty()) {
+					// flag has been set for guild
+					message.append(guildSetFlag.get(0).getValue() ? localEnabled.getValue() : localDisabled.getValue());
+				} else {
+					message.append(localDisabled.getValue());
+				}
+				message.append("\n");
+			});
```

Comment:
You won't have to do this anymore, because with the new approach we discussed all flags will be set for each guild, thus making this significantly easier :)